### ISSUE TITLE
Questions regading PowerHandlers

### ISSUE THREAD

**Issue by LordFokas** [BuildCraft/BuildCraft#1284]
I'm having some trouble with PowerHandlers, not exactly in using them, but with the way they work.

I believe there must be a great solid reason for this, so before anything, I'm asking for enlightenment.

We are forced to have power loss. At all times.
The PowerHandler and PowerReceiver final classes should be interfaces, to allow for more flexibility, including being able to merge several power systems in a single block more easily.

The PerditionCalculator, on the other hand, is public and non-final, and if we try to override every single thing in it, BuildCraft just seems to run some code behind our backs to make us lose even more power than we did by setting perdition to the minimum.

This is extremely annoying, especially when developing a storage unit where power is supposed to stay put until we say otherwise, and not leak before our very eyes.

So, to sum this up, the question is, why are we not free? Why must we always lose power? (just don't tell me because there's power loss everywhere IRL, Minecraft's logic and laws of physics are way incompatible with real life).

Thank you for reading the wall of text.
